DigiThinkIT/erpnext
# feat: Recruitment, Employee Life cycle, Holiday list and leave balance report fixes and enhancements

TODO:
- [x] Fixes
- [ ] Patches(Not required)
- [ ] Test Cases(Not required)

<h3>Job Opening</h3>
<h4>Traceback/Bug Report</h4> 

- The Job Opening was not getting created according to the staffing plan. Throwing "Job Openings for designation Manager already open or hiring completed as per Staffing Plan 2021"
- Department was not getting fetched from the staffing plan.

<h4>Steps to replicate</h4>

- The Job Opening was not getting created according to the staffing Plan
   * Create Staffing Plan For 2 developers.
   * Create job opening For 2 developers

<h3>Expected Behaviour/Fixes</h3>
Now job opening is as per Staffing plan. The behaviour is due to wrong if condition.
<hr>

<h3>Employee Onboarding</h3>

- Removed unused code for checking Unique Email and for Onboarding status.
- Employee Data is not being fetched if Employee is already Created.
- Fetch Designation and Department from Job title 

<h3>Employee separation</h3>

- Unused code for the status

<h3>Employee Leave Balance summary Report</h3>

- Added indentation for better readability.

<h3>Holiday List </h3>

- Setting notification off for Week off.
